docs/sql-ref-syntax-aux-resource-mgmt-add-jar.md

```diff
@@ -19,4 +19,31 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+`ADD JAR` adds a file, JAR or archive to the list of resources. The added file can be listed using [LIST JAR](sql-ref-syntax-aux-resource-mgmt-list-jar.html).
+
+### Syntax
+{% highlight sql %}
+ADD JAR file_name
+{% endhighlight %}
+
+### Parameters
+<dl>
+ <dt><code><em>file_name</em></code></dt>
+ <dd>The name of the file, JAR or archive to be added. It could be either on a local file system or a distributed file system.</dd>
```

Comment:
'archive' here means a .zip archive only, right, and only because a JAR file is a .zip file? if all of these things are just put on the classpath, I'd hesitate to say you can add a general 'archive' unless it has a special function beyond just treating it as a JAR file (and I don't know if it does). If it's just .zip files, I'd say "ZIP archive"